src/app/containers/Timestamp/index.jsx

```diff
@@ -1,86 +1,51 @@
-import moment from 'moment-timezone';
+import { number, string, bool } from 'prop-types';
 import Timestamp from '../../components/Timestamp';
 import relativeTime from './relativeTimestamp';
-import {
-  formatDateNumeric,
-  formatDate,
-  formatDateAndTime,
-  isValidDateTime,
-  formatUnixTimestamp,
-  isTenHoursAgoOrLess,
-} from './timestampUtilities';
-import { GridItemConstrainedMedium } from '../../lib/styledGrid';
+import { isValidDateTime, formatUnixTimestamp } from './timestampUtilities';
 
-const isSameDay = (dayToCompare, timestamp) => {
-  const day = moment(dayToCompare);
-  return day.isSame(timestamp, 'day');
-};
-
-const isToday = timestamp => isSameDay(Date.now(), timestamp);
-
-const formatType = timestamps => {
-  if (isToday(timestamps[0])) {
-    if (timestamps.length > 1 && !isSameDay(timestamps[1], timestamps[0])) {
-      return formatDate;
-    }
-    return formatDateAndTime;
-  }
-  return formatDate;
-};
-
-const humanReadable = ({
+const TimestampContainer = ({
   timestamp,
-  comparisonTimestamps,
-  shouldMakeRelative,
-}) =>
-  shouldMakeRelative
-    ? relativeTime(timestamp)
-    : formatUnixTimestamp(timestamp, formatType(comparisonTimestamps));
-
-const TimestampContainer = ({ lastPublished, firstPublished }) => {
-  if (
-    !isValidDateTime(new Date(lastPublished)) ||
-    !isValidDateTime(new Date(firstPublished))
-  ) {
+  dateTimeFormat,
+  format,
+  isRelative,
+  prefix,
+  suffix,
+  timezone,
+}) => {
+  if (!isValidDateTime(new Date(timestamp))) {
     return null;
   }
-
-  const firstPublishedString = humanReadable({
-    timestamp: firstPublished,
-    comparisonTimestamps: [firstPublished],
-    shouldMakeRelative:
-      lastPublished === firstPublished && isTenHoursAgoOrLess(firstPublished),
-  });
-
-  const lastPublishedString = `Updated ${humanReadable({
-    timestamp: lastPublished,
-    comparisonTimestamps: [lastPublished, firstPublished],
-    shouldMakeRelative: isTenHoursAgoOrLess(lastPublished),
-  })}`;
-
   return (
-    <GridItemConstrainedMedium>
-      <Timestamp
-        datetime={formatUnixTimestamp(firstPublished, formatDateNumeric)}
-      >
-        {firstPublishedString}
-      </Timestamp>
-      {lastPublished !== firstPublished ? (
-        <Timestamp
-          datetime={formatUnixTimestamp(lastPublished, formatDateNumeric)}
-        >
-          {lastPublishedString}
-        </Timestamp>
-      ) : null}
-    </GridItemConstrainedMedium>
+    <Timestamp
+      datetime={formatUnixTimestamp(timestamp, dateTimeFormat, timezone)}
+    >
+      {prefix ? `${prefix} ` : null}
+      {isRelative
+        ? relativeTime(timestamp)
+        : formatUnixTimestamp(timestamp, format, timezone)}
+      {suffix ? ` ${suffix}` : null}
+    </Timestamp>
   );
 };
 
 TimestampContainer.propTypes = {
-  firstPublished: number.isRequired,
-  lastPublished: number.isRequired,
+  timestamp: number.isRequired,
+  dateTimeFormat: string.isRequired,
+  format: string.isRequired,
```

Comment:
Strictly `format` shouldn't be required, as if `isRelative` is true, its not used. Can it default to `null` and do a check for it before it's used?